distribution-spec/v1.0/openregistry/junit.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="OCI Distribution Conformance Tests" tests="59" failures="0" errors="0" time="106.054">
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Populate registry with test blob" classname="OCI Distribution Conformance Tests" time="4.02818423"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Populate registry with test layer" classname="OCI Distribution Conformance Tests" time="6.576234994"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Populate registry with test manifest" classname="OCI Distribution Conformance Tests" time="4.784856368"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Get the name of a tag" classname="OCI Distribution Conformance Tests" time="5.121218909"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Get tag name from environment" classname="OCI Distribution Conformance Tests" time="0.000542962">
+        <skipped>/Users/guacamole/oss/distribution-spec/conformance/01_pull_test.go:82&#xA;you have skipped this test.&#xA;/Users/guacamole/oss/distribution-spec/conformance/setup.go:344</skipped>
```

Comment:
If you want, you can fuzz out the username used in this file (up to you either way)

Reply:
removed the skipping cased which removed the username thingy

Reply:
Please revert this change.. The test XML itself should not be modified. Just meant to say that you could replace "guacamole" in the output if you wanted to